packages/webclient/src/AuthHandler/IAuthorizationHandler.ts

```diff
@@ -2,4 +2,6 @@ export interface IAuthorizationHandler {
     getToken(force?: boolean, args?: string[]): Promise<string>;
     subscriptionID: string;
     baseUrl: string;
+    getCloudSuffixUrl?: (suffixName: string) => string;

```

Comment:
should we follow below syntax in parity with other function declarations:
getCloudSuffix?(suffixName: string): string;

Reply:
i have been trying to follow this approach:
propertyName: typeName
I will update getToken(force?: boolean, args?: string[]): Promise<string> to be 
getToken: (force?: boolean, args?: string[]) =>  Promise<string>